pulsar-broker/src/test/java/org/apache/pulsar/schema/compatibility/SchemaTypeCompatibilityCheckTest.java

```diff
@@ -55,7 +55,7 @@
     private static final String namespace = "test-namespace";
     private static final String namespaceName = PUBLIC_TENANT + "/" + namespace;
 
-    @BeforeClass
+    @BeforeMethod
```

Comment:
Why need to change to BeforeMethod?

Reply:
Each test should have its own Pulsar data.